Added VSCode debugging (for MvcSandbox) & code analysis support (#29486)
dotnet/aspnetcore: docs/BuildFromSource.md

This repo contains a Selenium-based tests require a version of Chrome to be installed. Download and install it from <https://www.google.com/chrome>.

#### Visual Studio Code Extension

> Following extension are recommanded for Visual Studio Code in order to work with ASP.NET Core repository.

Reviewer: ```suggestion
The following extensions are recommended when developing in the ASP.NET Core repository with Visual Studio Code.
```
